interface/orders/load_compendium.php

```diff
@@ -81,11 +81,18 @@ function getLabID($npi)
 
 <head>
 <?php Header::setupHeader(); ?>
-<title><?php echo xlt('Load Lab Configuration'); ?></title>
+<title><?php echo xlt('Load Compendium'); ?></title>
```

Comment:
Would change this to:
Load Lab Compendium

Reply:
Actually, this will make the tab to big. Leave this as is, but just change the page title below.